lang/funcs/filesystem.go

```diff
@@ -249,6 +258,13 @@ func MakeFileSetFunc(baseDir string) function.Function {
 					continue
 				}
 
+				// Remove the path and file separator from matches.
+				match = strings.TrimPrefix(match, path+string(filepath.Separator))
```

Comment:
The `filepath` package has a function [`Rel`](https://godoc.org/path/filepath#Rel) which implements this operation in a filesystem-path-aware way. I think it'd be more robust to use that function here unless there's a specific reason why it's inappropriate.

Reply:
Works well with `filepath.Rel()` 👍 Updated in 9eaf7eb9cbab64ec745cd5efbd3980cab10d2ac8